server/datastore/mysql/apple_mdm.go

```diff
@@ -5367,6 +5367,16 @@ VALUES (?, ?, ?, ?, ?, ?, ?, ?)
 	return tok, nil
 }
 
+func (ds *Datastore) CountABMTokens(ctx context.Context) (uint32, error) {
```

Comment:
Best to use `int` (signed) for things that could be manipulated with arithmetics (e.g. take that count, subtract another count, etc. and suddenly you get `18446744073709551606`. Unsigned is fine for ids but for anything that looks like stats we should use signed.

Reply:
E.g. https://go.dev/play/p/kA1DSC1EY1I

The fix of course is to convert to `int`, but that's an unnecessary footgun that's also hard to catch in review.